libbeat/scripts/Makefile

```diff
@@ -219,11 +219,11 @@ update: python-env
 
```

Comment:
There are two steps: Creating the environment and using it. To use it in the Makefile here it must be part of the same call, that is why I have to add it to each line. Probably there is a better solution?